omi-client/README.md

```diff
@@ -46,7 +51,106 @@ A `webpack.config.js` should include the following:
 
 ## Usage
 
-Forthcoming
+To submit items to the block chain, or read existing items value off the global
+state, you must create an `OmiClient`.  It can be created with a private key you
+can generate, or one that has been saved to disk.
+
+```
+let {OmiClient} = require('omi-client')
+let {signer} = require('sawtooth-sdk')
+
+const sawtoothBaseUrl = 'http://localhost:8080'
+const privateKey = signer.makePrivateKey() // or load existing key from some other source
+
+let client = new OmiClient(sawtoothBaseUrl, privateKey)
+```
+
+### Setting Objects
+
+Objects are set in the global state via a transaction submitted to the block
+chain. These objects are `IndividualIdentity`, `OrganizationalIdentity`,
+`Recording`, and `Work`. Each has a corresponding `set` method on the
+`OmiClient`: `setIndividual` for `IndividualIdentity`, and so on.
+
+Initial sets of an object result in a new object being stored in the global
+state.  Subsequent sets of an object result in an update operation.  These
+updates may only be performed by the owner of the private key that signed the
+original set operation.  In other words, ownership of the object is controlled
+by the public key associated with the private key used to create it.
+
+`OrganizationalIdentity` and `Recordings` both have type enumerations.  These
+can be accessed via
+
+```
+let {OrganizationalIdentity, Recording} = require('omi-client/protobuf')
+
+// and used like so (with an Organization example):
+
+client.setOrganization({
+  name: 'My Records',
```

Comment:
Maybe use a different example than 'My Records' since Record is an object type in the system.